client/src/templates/Challenges/classic/Show.js

```diff
@@ -249,6 +256,13 @@ class ShowClassic extends Component {
     );
   }
 
+  renderSpaceForFlashMessage = () => {
```

Comment:
I think it makes sense to put this everywhere that flash messages can go, rather than just challenges. Even though the other pages generally have space, it's probably best to have uniform behaviour.  @ahmadabdolsaheb what do you think?